src/main/java/org/ow2/proactive/procci/model/occi/infrastructure/mixin/Contextualization.java

```diff
@@ -34,7 +34,8 @@
      */
     public Contextualization(String userdata, List<Entity> entities) {
         super(Identifiers.INFRASTRUCTURE_SCHEME, Identifiers.CONTEXTUALIZATION, Identifiers.CONTEXTUALIZATION,
-                createAttributeSet(), new ArrayList<Action>(), new ArrayList<Mixin>(), setApplies(), entities);
+                createAttributeSet(), new ArrayList<Action>(), new ArrayList<Mixin>(), setApplies(),
```

Comment:
Why not using Lombok for getter and setter in this class? Would be convenient to be consistent.